src/it/java/teammates/it/ui/webapi/GetStudentActionIT.java

```diff
@@ -0,0 +1,155 @@
+package teammates.it.ui.webapi;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.Instructor;
+import teammates.storage.sqlentity.Student;
+import teammates.ui.output.StudentData;
+import teammates.ui.webapi.EntityNotFoundException;
+import teammates.ui.webapi.GetStudentAction;
+import teammates.ui.webapi.JsonResult;
+
+/**
+ * SUT: {@link GetStudentAction}.
+ */
+public class GetStudentActionIT extends BaseActionIT<GetStudentAction> {
+
+    @Override
+    @BeforeMethod
+    protected void setUp() throws Exception {
```

Comment:
What would be the purpose to throw a generic exception here and a few other places instead of something more specific?